web/src/features/App/api/useLicense.ts

```diff
@@ -26,7 +26,7 @@ function useLicenseWithIntercept() {
         return x;
       },
       (error) => {
-        if (error.response.status === 401) {
+        if (error?.response?.status=== 401) {
```

Comment:
was getting a lot of console log noise from this from the cors errors. I think I fixed all of them.